MAUI/TreeMap/selection-and-highlights.md

```diff
@@ -0,0 +1,548 @@
+/// </summary>
+public class PopulationDetails
+{
+    public string Country { get; set; }
+    public string Continent { get; set; }
+    public int Population { get; set; }
+}
+
+{% endhighlight %}
+{% highlight C# tabtitle="PopulationViewModel.cs" %}
+
+/// <summary>    
+/// Represents demographic information for countries in the Americas for the year 2023.
+/// </summary>
+public class PopulationViewModel
+{
+    public PopulationViewModel()
+    {
+        this.PopulationDetails = new ObservableCollection<PopulationDetails>()
+        {
+            new PopulationDetails() { Continent ="North America", Country = "United States of America", Population = 339996564 },
+            new PopulationDetails() { Continent ="South America", Country = "Brazil", Population = 216422446 },
+            new PopulationDetails() { Continent ="North America", Country = "Mexico", Population = 128455567 },
+            new PopulationDetails() { Continent ="South America", Country = "Colombia", Population = 52085168 },
+            new PopulationDetails() { Continent ="South America", Country = "Argentina", Population = 45773884 },
+            new PopulationDetails() { Continent ="North America", Country = "Canada", Population = 38781292 },
+            new PopulationDetails() { Continent ="South America", Country = "Peru", Population = 34352719 },
+            new PopulationDetails() { Continent ="South America", Country = "Venezuela", Population = 28838499 },
+            new PopulationDetails() { Continent ="South America", Country = "Chile", Population = 19629590 },
+            new PopulationDetails() { Continent ="South America", Country = "Ecuador", Population = 18190484 },
+            new PopulationDetails() { Continent ="North America", Country = "Guatemala", Population = 18092026 },
+            new PopulationDetails() { Continent ="South America", Country = "Bolivia", Population = 12388571 },
+            new PopulationDetails() { Continent ="North America", Country = "Honduras", Population = 10593798 },
+            new PopulationDetails() { Continent ="North America", Country = "Nicaragua", Population = 7046311 },
+            new PopulationDetails() { Continent ="South America", Country = "Paraguay", Population = 6861524 },
+            new PopulationDetails() { Continent ="North America", Country = "El Salvador", Population = 6364943 },
+            new PopulationDetails() { Continent ="North America", Country = "Costa Rica", Population = 5212173 },
+            new PopulationDetails() { Continent ="South America", Country = "Uruguay", Population = 3423109 },
+        };
+    }
+
+    public ObservableCollection<PopulationDetails> PopulationDetails
+    {
+        get;
+        set;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Multiple selection
+With `Multiple` selection mode you can select multiple items simultaneously, facilitating comparative analysis and exploration of related data within the Treemap.
```

Comment:
In the Multiple selection mode, users can select multiple items simultaneously within the Treemap control.